packs/tfe_fdo_nomad/outputs.tpl

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+Congrats! You deployed the terraform_enterprise_fdo pack on Nomad.    
+
+You can view your instances of TFE running on the Nomad UI and reach it on the provided hostname.
```

Comment:
It seems like there's something missing in this Pack around how we expect the user to have established a hostname-to-IP address mapping.

Reply:
@tgross This is a hard requirement to run TFE, users need to have a `Hostname with end-user routable DNS` (https://developer.hashicorp.com/terraform/enterprise/flexible-deployments/install/requirements).

Reply:
The README for this pack doesn't include any references to those docs. A Pack should be as self-contained as possible, which means at least having links to those docs.